sky/packages/sky_engine/LICENSE

```diff
@@ -2170,37 +2170,6 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
 --------------------------------------------------------------------------------
 boringssl
```

Comment:
this still contains changes to licenses for code you didn't change

Reply:
I could revert the file but this is the result of running the licenses scripts as described in tools/licenses/README.md.  If those directions aren't correct they should be updated

Reply:
we should figure out when that file diverged from the .golden files, and have whoever forgot to update the LICENSE file then (and whoever reviewed the PR in question) fix it, because otherwise that person will continue not knowing that they should have updated the LICENSE file.

Reply:
According to the git logs the last time someone besides the skia autoroller updated that file was https://github.com/flutter/engine/commit/4a4cff96d472416dd86778c8a91d318dc8f26664.  There is a long list of commits since then (mostly skia autorolls), I have a simple script to filter down the rest and I can bisect from there .  There was also changes to the license script since then, it is possible that those changes cause the produced file to be different for the same inputs

Reply:
I can go back as far as 645c15fd2 before gclient sync stops working:

```
Package doesn't exist (the pubspec for front_end 0.1.0 from path has version 0.1.0-alpha.12).
Error: Command '../../../src/third_party/dart/tools/sdks/mac/dart-sdk/bin/pub get' returned non-zero exit status 69 in /Users/jonahwilliams/Documents/Flutter/engine/src/flutter/frontend_server/
```

But even 7e145b1 had a diff in the LICENSE file when I ran the script.

Reply:
Any chance we could catch a missed update to the LICENSE in a presubmit check on Travis?

Reply:
Filled https://github.com/flutter/flutter/issues/19048

Reply:
I'm not sure what else is actionable here.  However we go about making sure our license files get updated shouldn't effect this PR